posts/inside-rust/2023-09-19-crates-io-usage-policy-rfc.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: crates.io Policy Update RFC
+author: Tobias Bieniek
+team: the crates.io team <https://www.rust-lang.org/governance/teams/crates-io>
+---
+
+Around the end of July the crates.io team opened an [RFC](https://github.com/rust-lang/rfcs/pull/3463) to update the current crates.io usage policies. This update addresses operational concerns that have arisen since the last significant update in 2017, particularly related to name squatting and spam. This has caused considerable discussion, and most of the suggested improvements have since been integrated into the proposal.
```

Comment:
I feel a little unclear about what these sentences are referring to, exactly. I think this would help:

```suggestion
Around the end of July the crates.io team opened an [RFC](https://github.com/rust-lang/rfcs/pull/3463) to update the current crates.io usage policies. This policy update addresses operational concerns of the crates.io community service that have arisen since the last significant policy update in 2017, particularly related to name squatting and spam. The RFC has caused considerable discussion, and most of the suggested improvements have since been integrated into the proposal.
```

Reply:
sounds good to me, thanks!